CMakeLists.txt

```diff
@@ -352,8 +355,8 @@ else()
 
   # Setup python directories for plugin processes in same style as KWIVER
   if( VIAME_ENABLE_PYTHON )
-    set( kwiver_python_subdir "python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" )
-    set( kwiver_python_output_path "${VIAME_BUILD_PREFIX}/lib/${kwiver_python_subdir}" )
+    pycmd(python_site_packages "from distutils import sysconfig; print(sysconfig.get_python_lib(prefix=''))")
+    set( kwiver_python_output_path "${VIAME_BUILD_PREFIX}/${python_site_packages}" )
```

Comment:
This makes the path on windows odd lib/Lib/python...

Reply:
`sysconfig.get_python_lib(prefix='')` should return `Lib\\site-packages` on windows. Is that not happening? What happens when you message the `python_site_packages` var?

Reply:
It puts the files in [install]/lib/Lib/... shouldn't there only be a single lib?